pkg/JuliaInterface/src/JuliaInterface.c

```diff
@@ -16,7 +16,7 @@
 #include "convert.h"
 #include "sync.h"
 
-#include <julia_gc.h>    // GAP header
+#include <gap/julia_gc.h>   // GAP header
```

Comment:
Note that this will only work with an "installed" GAP, not with an uninstalled GAP. This shouldn't currently be a problem because of course when using the JLLs, GAP is installed; and also `etc/setup_override_dir.jl` does `make install`.

So, no objection here.